packages/tokens/icon-tokens/tokens.json

```diff
@@ -0,0 +1,41 @@
+{
+  "nl": {
+    "icon": {
+      "block-size": {
+        "$extensions": {
+          "nl.nldesignsystem.css-property-syntax": "<length>",
+          "nl.nldesignsystem.figma-implementation": true
+        },
+        "$type": "dimension"
+      },
+      "color": {
+        "$extensions": {
+          "nl.nldesignsystem.css-property-syntax": "<color>",
+          "nl.nldesignsystem.figma-implementation": true
+        },
+        "$type": "color"
+      },
+      "font-size": {
+        "$extensions": {
+          "nl.nldesignsystem.css-property-syntax": ["<length>", "<percentage>"],
+          "nl.nldesignsystem.figma-implementation": true
+        },
+        "$type": "fontSizes"
+      },
+      "inline-size": {
+        "$extensions": {
+          "nl.nldesignsystem.css-property-syntax": "<length>",
+          "nl.nldesignsystem.figma-implementation": true
+        },
+        "$type": "dimension"
+      },
+      "inset-block-start": {
+        "$extensions": {
+          "nl.nldesignsystem.css-property-syntax": "<length>",
+          "nl.nldesignsystem.figma-implementation": false
```

Comment:
@jeffreylauwers ik heb hier `false` van gemaakt omdat je dit volgens mij in Figma moet faken.